apps/challenges/aws_utils.py

```diff
@@ -747,20 +812,22 @@ def get_logs_from_cloudwatch(log_group_name, log_stream_prefix, start_time, end_
     """
     client = get_boto3_client("logs", aws_keys)
     logs = []
-    try:
-        response = client.filter_log_events(
-            logGroupName=log_group_name,
-            logStreamNamePrefix=log_stream_prefix,
-            startTime=start_time,
-            endTime=end_time,
-            filterPattern=pattern
-        )
-        for event in response["events"]:
-            logs.append(event["message"])
-    except ClientError as e:
-        logger.exception(e)
-        return ["There was an error in displaying the logs."]
-
+    if ENV == 'dev':
+        logs = ["Sample Log. Can't fetch Cloudwatch logs in test/dev environment."]
+    else:
+        try:
+            response = client.filter_log_events(
+                logGroupName=log_group_name,
+                logStreamNamePrefix=log_stream_prefix,
+                startTime=start_time,
+                endTime=end_time,
+                filterPattern=pattern
+            )
+            for event in response["events"]:
+                logs.append(event["message"])
+        except Exception as e:
+            logger.exception(e)
+            return ["There was an error in displaying the logs."]
```

Comment:
`[f"There is an error in displaying logs. Please find the full error traceback here {e}"]`